### ISSUE TITLE
Flatpak packaging

### ISSUE THREAD

**Issue by curiouslad** [httpie/desktop#170]
Flatpak is getting more and more popular and is in my opinion becoming standard for packaging applications for linux. I could probably make manifest.json and create github action for publishing to flathub. Let me know if this is something you would be interested in.

**Comment by skaldebane** [comment]
Would love this. I actually discovered HTTPie through an unofficial Flatpak package on Flathyb, so it's a great way to make it discoverable by new users.

Unfortunately that unofficial package no longer seems to be updated, and is stuck on a version from 1 year ago.

Technically this shouldn't be too hard, since you already have an AppImage, which you can easily embed in a Flatpak. See https://github.com/flathub/sh.ppy.osu for an example of this from the osu! Flatpak.

**Comment by neufeind** [comment]
Official flatpaks would still be great. But I wanted to let you know the unofficial package was just updated - current version and also using an updated freedesktop platform.

**Comment by yermulnik** [comment]
> But I wanted to let you know the unofficial package was just updated

This one at https://flathub.org/en/apps/io.httpie.Httpie ?

**Comment by neufeind** [comment]
Yes, that's the current release but as the list shows it was recently built. So it seems to at least have dropped the end-of-life platform packages and was rebuilt.
